robots/commenter/main.go

```diff
@@ -209,13 +216,31 @@ func makeCommenter(comment string, useTemplate bool) func(meta) (string, error)
 	}
 }
 
-func run(c client, query, sort string, asc, random bool, commenter func(meta) (string, error), ceiling int) error {
-	log.Printf("Searching: %s", query)
-	issues, err := c.FindIssues(query, sort, asc)
-	if err != nil {
-		return fmt.Errorf("search failed: %v", err)
+func run(c client, queries, sort string, asc, random bool, commenter func(meta) (string, error), ceiling int, labelsUpdated time.Duration) error {
+	var issues []github.Issue
+	for _, query := range strings.Split(queries, querySeparator) {
+		log.Printf("Searching: %s", query)
+		foundIssues, err := c.FindIssues(query, sort, asc)
+		if err != nil {
+			return fmt.Errorf("search failed: %v", err)
+		}
+		for _, foundIssue := range foundIssues {
+			var alreadyInIssues bool
+			for _, issue := range issues {
+				if foundIssue.ID == issue.ID {
+					alreadyInIssues = true
+					break
+				}
+			}
+			if !alreadyInIssues {
+				if labelsUpdated != 0 && time.Now().Add(-labelsUpdated).Before(foundIssue.UpdatedAt) {
```

Comment:
Is it really that easy to check if the label had not changed for that timeframe?
I would have thought we have to go through the changes of the PR to see if the labels stayed the same for a certain time